tools/moFF/test-data/output1/moff_mbr_test_pepsum.tab

```diff
@@ -0,0 +1 @@
+peptide	prot	sumIntensity_mbr_test2	sumIntensity_mbr_test1
```

Comment:
This file is now not used anymore, right?

Reply:
true! removing now